pytket/conanfile.py

```diff
@@ -37,7 +37,7 @@ def requirements(self):
         self.requires("nanobind/tci-2.8.0@tket/stable")
         self.requires("symengine/tci-0.14.0.1@tket/stable")
         self.requires("tkassert/0.3.4@tket/stable")
-        self.requires("tket/2.1.37@tket/stable")
+        self.requires("tket/2.1.38@tket/stable")
```

Comment:
After #1988 (just merged), this information now goes into `TKET_VERSION`, not in these two conanfiles.